src/components/geometry.js

```diff
@@ -9,10 +12,12 @@ var dummyGeometry = new THREE.Geometry();
  * Extended with registered geometries.
  */
 module.exports.Component = registerComponent('geometry', {
+  dependencies: ['position', 'rotation'],
```

Comment:
newline after this.

is this needed? position/rotation are default components. are they being initialized after?